pkg/README.md

```diff
@@ -1,28 +1,24 @@
-# Packaging for Bazel
-
-<div class="toc">
-  <h2>Rules</h2>
-  <ul>
-    <li><a href="#pkg_tar">pkg_tar</a></li>
-    <li><a href="#pkg_zip">pkg_zip</a></li>
-    <li><a href="#pkg_deb">pkg_deb</a></li>
-    <li><a href="#pkg_rpm">pkg_rpm</a></li>
-  </ul>
-</div>
+# rules_pkg - Archive building rules
 
+-   [Release Notes](#notes)
+-   [Overview](#overview)
+-   [Roadmap](#roadmap)
+-   [Reference](docs/reference.md)
+-   [Examples](examples/readme.md)
 
+<a name="notes"></a>
 ## Release Notes
 
 Version 0.3.0 or later requires Bazel 3.2.0 or later.
 
-
```

Comment:
Perhaps instead of (or in addition to) this, we could have a "requirements" section that mentions what tools need to be present?  `rpmbuild` for `pkg_rpm` comes to mind.

If you don't want to do this now, I can add one after this is committed.

Reply:
Great idea. I just don't know how to say it now. Let's do a PR just for that, because there are so many ways to slice and dice requirements which only impact some rules and the matrix of rules version + bazel version. This, in turn, becomes interesting if we version the rules with branches, so the matrix vanishes into branched sources.